modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/CmgRaftService.java

```diff
@@ -338,10 +338,12 @@ public CompletableFuture<Void> updateLearners(long term) {
 
         if (newLearners.isEmpty()) {
             // Methods for working with learners do not support empty peer lists for some reason.
-            return raftService.changePeersAndLearnersAsync(newConfiguration, term)
+            // TODO: https://issues.apache.org/jira/browse/IGNITE-26855.
+            return raftService.changePeersAndLearnersAsync(newConfiguration, term,  0)
```

Comment:
Below I did suggest/ask why it's '>' and not '>=' in `this.conf.getConf().getSequenceToken() > newConf.getSequenceToken()`. In case of proposed substitution CMG/MG reconfigurations will break.